tests/sentry/api/endpoints/test_organization_projects.py

```diff
@@ -182,6 +182,20 @@ def test_user_projects(self):
         # Verify projects that were returned were foo_users projects
         self.check_valid_response(response, foo_user_projects)
 
+
+class OrganizationProjectsCountTest(APITestCase):
+    @fixture
+    def org(self):
+        return self.create_organization(owner=self.user, name="baz")
```

Comment:
The base tests cases have an existing factory property for `self.organization` and `self.team` do you need to redefine them?

Reply:
Oh, good catch updated `OrganizationProjectsTest` too :+1: